### PR TITLE
Add PublicMint mainNet and testNet

### PR DESCRIPTION
**Pull request by rafaeltorres-seegno**
Added PublicMint mainNet (2020) and testNet (2019) contract addresses

### CODE REVIEW

**Comment by moisses89** [comment]
Should be listed on chainlist, and AFAIK 2020 chainId is other chain https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-2020.json

**Comment by rafaeltorres-seegno** [comment]
@moisses89 the chains have been added / updated on ethereum-lists/chains. You can continue with the review.

File: src/assets/v1.3.0/sign_message_lib.json [hunk 1]
```diff
@@ -52,6 +52,8 @@
     "1984": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "2001": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "2008": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
+    "2019": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
+    "2020": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
```
Comment (moisses89) [inline src/assets/v1.3.0/sign_message_lib.json:56]: Seems not be deployed 
```
reusing "MultiSendCallOnly" at 0x40A2aCCbd92BCA938b02010E17A5b8929b49130D
deploying "SignMessageLib"
```
If I request the bytecode to the rpc, I receive that is empty.

```
curl --location --request POST 'https://rpc.publicmint.io:8545' \
--header 'Content-Type: application/json' \
--data-raw '{
    "jsonrpc": "2.0",
    "method": "eth_getCode",
    "params": [
        "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
        "latest"
    ],
    "id": 1}'
```
Result empty:
```
 {
  "jsonrpc" : "2.0",
  "id" : 1,
  "result" : "0x"
}
```

**Review by moisses89 — CHANGES_REQUESTED** [review]

File: src/assets/v1.3.0/sign_message_lib.json (hunk 1 above)
Comment (rafaeltorres-seegno) [reply src/assets/v1.3.0/sign_message_lib.json:56]: Missing contract has been deployed.

**Review by moisses89 — APPROVED** [review]
Looks good to me!

**Pull request merged**